blocks/embed/embed.js

```diff
@@ -41,7 +47,7 @@ const embedVimeo = (url) => {
 };
 
 const embedTwitter = (url) => {
-  const embedHTML = `<blockquote class="twitter-tweet"><a href="${url}"></a></blockquote>`;
+  const embedHTML = `<blockquote class="twitter-tweet"><a href="${encodeURI(url)}"></a></blockquote>`;
```

Comment:
I don't think this is correct. you'd need a `encodeForHTMLAttribute()`,
https://cheatsheetseries.owasp.org/cheatsheets/Cross_Site_Scripting_Prevention_Cheat_Sheet.html#output-encoding-for-html-attribute-contexts